### PR TITLE
Fix issues in ps directory

### PR DESCRIPTION
**Pull request by saundefined**

### CODE REVIEW

File: reference/ps/functions/ps-setcolor.xml [hunk 1]
```diff
@@ -100,21 +100,23 @@
   </para>
  </refsect1>
 
- <refsect1>
-  <title>Bugs</title>
```
Comment (afilina) [inline reference/ps/functions/ps-setcolor.xml:104]: Perhaps I'm misunderstanding what this PR is meant to accomplish, but shouldn't the Bugs section appear somewhere else in the diff?

Comment (cmb69) [reply reference/ps/functions/ps-setcolor.xml:104]: The bugs section has been renamed (and moved a few lines down) to the usual notes section. LGTM.

File: reference/ps/functions/ps-setcolor.xml [hunk 2]
```diff
@@ -100,21 +100,23 @@
...
+
+ <refsect1 role="notes">
+  &reftitle.notes;
+  <note>
+   <para>
+    The second parameter is currently not always evaluated. The color is
+    sometimes set for filling and drawing just as if
+    <literal>fillstroke</literal> were passed.
+   </para>
+  </note>
```
Comment (cmb69) [inline reference/ps/functions/ps-setcolor.xml:118]: Maybe:
```suggestion
  <caution>
   <para>
    The second parameter is currently not always evaluated. The color is
    sometimes set for filling and drawing just as if
    <literal>fillstroke</literal> were passed.
   </para>
  </caution>
```

Comment (saundefined) [reply reference/ps/functions/ps-setcolor.xml]: Thanks! Looks good

File: reference/ps/functions/ps-show-boxed.xml [hunk 3]
```diff
@@ -120,163 +120,163 @@
...
+      <listitem>
+       <para>Distance between baselines of two consecutive lines.</para>
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term>linebreak (parameter)</term>
+      <listitem>
+       <para>
+        Set to "true" if you want a carriage return to start a new line instead
+        of treating it as a space. Defaults to "false".
```
Comment (zors1) [inline reference/ps/functions/ps-show-boxed.xml:144]: Shouldn't "true" and "false" be replaced with `&true;` and `&false;`?

File: reference/ps/functions/ps-show-boxed.xml [hunk 4]
```diff
@@ -120,163 +120,163 @@
...
+      </listitem>
+     </varlistentry>
+     <varlistentry>
+      <term>parindentskip (value)</term>
+      <listitem>
+       <para>
+        Number of paragraphs in the box whose first lines will not be indented.
+        This defaults to 0. This is useful for paragraphs right after a section
+        heading or text being continued in a second box. In both case one would
+        set this to 1.
```
Comment (zors1) [inline reference/ps/functions/ps-show-boxed.xml:220]: And shouldn't integers in this section be wrapped in `<literal>`?

File: reference/ps/functions/ps-show-boxed.xml (hunk 3 above)
Comment (cmb69) [reply reference/ps/functions/ps-show-boxed.xml:144]: Almost certainly. The actual string parameters below could be marked up as `<literal>`.

**Review by afilina — APPROVED** [review]

**Pull request merged**